terasoluna-gfw-functionaltest-web/src/main/java/org/terasoluna/gfw/functionaltest/app/codepoints/CodepointsController.java

```diff
@@ -0,0 +1,247 @@
+/*
+ * Copyright (C) 2013-2016 terasoluna.org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+package org.terasoluna.gfw.functionaltest.app.codepoints;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.terasoluna.gfw.common.codepoints.CodePoints;
+import org.terasoluna.gfw.common.codepoints.catalog.JIS_X_0208_Katakana;
+
+@Controller
+@RequestMapping(value = "codepoints")
+public class CodepointsController {
+	
+	@RequestMapping(method = RequestMethod.GET)
+	public String index() {
+		
+		return "codepoints/index";
+	}
+	
+	@RequestMapping(value="01", method = RequestMethod.GET)
+	public String codepoints01(Model model) {
+		
+		CodePoints codepoints1 = CodePoints.of(JIS_X_0208_Katakana.class);
+		CodePoints codepoints2 = CodePoints.of(JIS_X_0208_Katakana.class);
+
+		model.addAttribute("instanceCheck", codepoints1.equals(codepoints2));
+		model.addAttribute("containsAll", codepoints1.containsAll("カナ"));
+		
+		return "codepoints/result_containsAll";
+	}
+	
+	@RequestMapping(value="02", method = RequestMethod.GET)
+	public String codepoints02(Model model) {
+		
+		CodePoints codepoints = new JIS_X_0208_Katakana();
+		
+		model.addAttribute("containsAll", codepoints.containsAll("カナ"));
+		
+		return "codepoints/result_containsAll";
+	}
+	
+	@RequestMapping(value="03", method = RequestMethod.GET)
+	public String codepoints03(Model model) {
```

Comment:
- [ ] codepointsXX should be implemented at once and inputted data from selenium because the logics are same.